helpers/build_packages.sh

```diff
@@ -328,22 +331,21 @@ if [ "$BUILDGG" = "1" ]; then
         minfo "Not building audioflingerglue and pulseaudio-modules-droid-glue due to pulseaudio-modules-droid-hidl in patterns"
     elif grep -qs "- pulseaudio-modules-droid-glue" "$pattern_lookup" ||
          grep -qs "Requires: pulseaudio-modules-droid-glue" "$metapackage_lookup"; then
-        audioflingerglue_version=$(git --git-dir external/audioflingerglue/.git describe --tags 2>/dev/null | sed -r "s/\-/\+/g")
+        pkg=audioflingerglue
+        cd external/$pkg || die "Could not change directory to external/$pkg"
+        audioflingerglue_version=$(get_package_version "$pkg")
         if [ -z "$audioflingerglue_version" ]; then
```

Comment:
get_package_version() already exits on failure.

Reply:
`get_package_version()` is called in a sub-shell, thus exit just exits that sub-shell.

Reply:
ahh, you're right.